code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -358,11 +361,17 @@
 /datum/reagent/slimejelly/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
 	if(alien == IS_DIONA)
 		return
-	if(prob(10))
+	if(prob(10) && alien != IS_SLIME) //Partially made of the stuff. Why would it hurt them?
```

Comment:
So rather than doing this, put the IS_SLIME check under the IS_DIONA check and put the normal checks in a else?